server/service/src/sync/translations/program_requisition_settings.rs

```diff
@@ -29,6 +29,7 @@ pub struct LegacyListMasterRow {
     #[serde(rename = "programSettings")]
     program_settings: Option<LegacyProgramSettings>,
     is_immunisation: Option<bool>,
+    inactive: Option<bool>,
```

Comment:
I don't think this or `is_immunisation` would be `Option`? Hmm issue doesn't have much detail 😆  https://github.com/msupply-foundation/open-msupply/issues/2534. But from what I have gleaned it's to keep backwards compatibility if you're syncing OMS on an old version of OG central server. Fair enough.

Reply:
Yes that's why it's optional (: